index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Hello Microverse</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="logo">My Logo</div>
+            <div class="onion">
+                <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd"
+                        d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
+                        fill="#6070FF" />
+                </svg>
+
+            </div>
+        </header>
+        <main>
+            <div class="intro primary-text1">
+                I am Dagmawi, Glad to see you
+            </div>
+            <div class="intro primary-text2">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="intro connect">
+                LET'S CONNECT
+            </div>
```

Comment:
- [ ] For all your buttons, please make sure to use either an `<a>` or an `<button>` one instead of a `<div>`. 
- [ ] Also, please make sure to implement the _hover, active, and pressed_ properties in all your buttons. This is because this helps the user to navigate through your website without problems.